bundles/org.openhab.binding.unifi/README.md

```diff
@@ -287,3 +287,90 @@ sitemap unifi label="UniFi Binding"
+
+The `generateVoucher(s)` actions parameters match the configuration parameters for the `guestVouchersGenerate` channel.
+With the actions, these parameters can be controlled in a rule or script.
+`null` values for the parameters are allowed, and will set the parameter to the default value.
+
+| Parameter     | Description                                                                 | Default |
+| ------------- | --------------------------------------------------------------------------- | ------- |
+| count         | Number of vouchers to create                                                |    1    |
+| expire        | Minutes a voucher is valid after activation (default is 1 day)              |  1440   |
+| users         | Number of users for voucher, 0 for no limit                                 |    1    |
+| upLimit       | Upload speed limit in kbps, no limit if not set                             |         |
+| downLimit     | Download speed limit in kbps, no limit if not set                           |         |
+| dataQuota     | Data transfer quota in MB per user, no limit if not set                     |         |
+
+The `revoke...` actions allow you to revoke previously created vouchers.
+The parameter is the voucher code or a list of voucher codes to be revoked.
+
+The `listVouchers` action will return a json string representing the currently available vouchers for the site.
+The json contains all parameters for the voucher, therefore it is possible to filter on these in a rule or script.
+For example, one could retrieve all vouchers created before a certain time and use the `revokeVouchers` action to delete these.
+The structure of the returned json is (depending on content, some fields may be missing):
+
+```json
+[
+    {
+        "code": "3867791284",
+        "createTime": "2023-01-31T14:40:47Z",
+        "duration": 1440,
+        "quota": 2,
+        "used": 0,
+        "qosUsageQuota": 300,
+        "qosRateMaxUp": 200,
+        "qosRateMaxDown": 100,
+        "qosOverwrite": true,
+        "note": "I added a note when creating vouchers in the UniFi hotspot UI",
+        "status": "VALID_MULTI"
+    },
+    {
+        "code": "0021952641",
+        "createTime": "2023-01-31T14:38:47Z",
+        "duration": 1440,
+        "quota": 1,
+        "used": 0,
+        "qosOverwrite": false,
+        "status": "VALID_ONE"
+    },
+    { ... }
+]
+```
+
+Examples:
+
+```java
+val uniFiActions = getActions("unifi","unifi:site:mycontroller:mysite")
+val success = uniFiActions.generateVoucher(null, null, null, 100, 500, 250)
+```
+
+```java
+val uniFiActions = getActions("unifi","unifi:site:mycontroller:mysite")
+val vouchersJson = uniFiActions.listVouchers()
+```
+
+```java
+import java.util.List
+
+val List<String> voucherList = newArrayList("38677-91284", "46415-36104")
+val uniFiActions = getActions("unifi","unifi:site:mycontroller:mysite")
+val success = uniFiActions.revokeVouchers(voucherList)
+```
```

Comment:
This could be moved to separate sub chapter under "Full Example".

Reply:
Done.